### DISCUSSION TITLE
Unable to Acquire Fluxite in Mods

### DISCUSSION THREAD

**Discussion by munanshang** [Rearth/Oritech#54] [Q&A]
Fluxite is an essential mineral required in the mid to late stages of the game. Currently, there is only one machine that can process it, but the components of this machine also require Fluxite to be crafted.

**Comment by Rearth** [comment]
Did you notice the tooltip of fluxite? There's also a manual entry link when you have the manual on you. The primary way of earning fluxite is with an enderic laser targeting amethyst buds. For the enderic laser you don't need any fluxite. Please let me know if this solves your issue

**Reply by munanshang** [reply]
Yes, the problem has been resolved. I apologize for overlooking this part of the content due to the lack of Chinese support. I couldn't find the relevant formula in the synthesis table query.

However, another issue has been discovered; the textures of these blocks have problems in version 1.21.
![PixPin_2024-08-03_18-31-51](https://github.com/user-attachments/assets/8ce663f1-2424-4087-bd0a-007e0f931e3e)

**Reply by Rearth** [reply]
Ah yeah, I can imagine the mod being hard to understand. Maybe at some point a chinese translation will be available.

**Reply by Rearth** [reply]
Regarding the issue you're showing: make sure you have the athena mod installed. If you have it installed and still have issues, please check if you have sodium installed. The rendering will break when sodium without indium is installed, so install indium aswell

**Comment by Rearth** [comment]
I'll mark this as closed for now. If you have any more questions or issues, feel free to reopen this
